src/main/java/me/jellysquid/mods/sodium/mixin/features/fast_leaf_culling/MixinLeavesBlock.java

```diff
@@ -0,0 +1,29 @@
+package me.jellysquid.mods.sodium.mixin.features.fast_leaf_culling;
+
+import me.jellysquid.mods.sodium.client.SodiumClientMod;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.LeavesBlock;
+import net.minecraft.block.Material;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.option.GraphicsMode;
+import net.minecraft.util.math.Direction;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(LeavesBlock.class)
+public class MixinLeavesBlock extends Block {
+    public MixinLeavesBlock() {
+        super(Settings.of(Material.AIR));
+        throw new AssertionError("Mixin constructor called!");
+    }
+
+    @Override
+    public boolean isSideInvisible(BlockState state, BlockState stateFrom, Direction direction) {
```

Comment:
`@Override` -> `@Overwrite` with reason, please. This makes it more explicit what's going on.

Reply:
The problem is, there's no `isSideInvisible` to overwrite in `LeavesBlock`

Reply:
Ah... you're right. That's my fault. Apologies.